packages/account/src/Configs/get-status-badge-config.tsx

```diff
@@ -42,10 +42,10 @@ const getStatusBadgeConfig = (
             return {
                 text: (
                     <Localize
-                        i18n_default_text='<0>Need verification.</0><1>Verify now</1>'
+                        i18n_default_text='<0>Needs verification.</0><1>Verify now</1>'
                         components={[
                             <Text key={0} weight='bold' size='xxxs' color='var(--status-info)' />,
-                            <Link key={1} className='link-need-verification' to='/account/proof-of-identity' />,
+                            <Link key={1} className='link-need-verification' to='/account/proof-of-identity' />, // TODO: [deriv-eu] change this link when we have the new verification pop-up
```

Comment:
Check on this: needs to be remove

Reply:
need to add condition to redirect to POI or POA

Reply:
is this done?